docs/quickstart/quickstart.ipynb

```diff
@@ -73,11 +73,13 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
```

Comment:
why did we change the `quickstart.ipynb` - that seems strange because it has nothing to do with logging. Can you remove that changes.